ideasbox/library/forms.py

```diff
@@ -4,15 +4,24 @@
 
 from .models import Book, BookSpecimen
 from .utils import fetch_from_openlibrary, load_from_moccam_csv, load_unimarc
+from  django.core.exceptions import ValidationError
 
 
 class BookSpecimenForm(forms.ModelForm):
+    
+    def clean_spfile(self):
+        # Does not allow to enter a file and a serial
+        if (self.cleaned_data['spfile']!=None)and(self.cleaned_data.get('serial')!=''):
+            raise ValidationError('You entered a file and a serial')
```

Comment:
Every string that could be seen by a end user need to be translatable, so using gettext (just look other files around to see how it's used).